.changeset/tough-ears-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": patch
+---
+
+feat: add new icons
```

Comment:
I feel it would be good to mention which icons have been added.

Reply:
Makes sense, added to the PR title and desc.